io.catenax.part_as_planned/1.0.1/metadata.json

```diff
@@ -1 +1 @@
-{ "status" : "release"}
+{ "status" : "standardize"}
```

Comment:
All the models which are going to be standardize, we need to convert them to SAMM model. as this model is still written in BAMM, can you please change this to SAMM and raise new PR.